File: Data Structures & Algorithms/sliding-window-maximum/submission-3.cpp
class Solution {
public:
    vector<int> maxSlidingWindow(vector<int>& nums, int k) {
         if(k==1) return nums;
         int n=nums.size();
         int i=0;
         int j=k-1;
         vector<int>ans;
         int maxi=INT_MIN;
        
        for(int i=0;i<=n-k;i++){
          int maxi=nums[i];
          for(int j=i;j<i+k;j++){
             if(nums[j]>maxi){

                maxi=max(maxi,nums[j]);
             }


          }
          ans.push_back(maxi);



        }












        return ans;




    }
};

File: Data Structures & Algorithms/remove-element/submission-3.java
class Solution {
    public int removeElement(int[] nums, int val) {
        
        // int i=0,j=1;
        // if(nums.length==0) return 0;  

        // while(i<nums.length&&j<nums.length){

        //   if(nums[i]==val&&nums[j]!=val){
        //         nums[i]=nums[j];
        //         nums[j]=val;
        //         i++;
        //         j++;
        //   }
        //   else if(nums[i]!=val&&nums[j]!=val){
        //       i++;
        //       j++;
        //   }
        //   else if(nums[i]!=val&&nums[j]==val){
        //     i++;
        //   }
        //   else if(nums[i]==val&&nums[j]==val){
        //     j++;
        //   }

        // }


        // for(int ik=0;ik<nums.length;ik++){
        //   if(nums[ik]==val) return ik;
        //   System.out.print(nums[ik]);
        // }
       
        // return nums.length;


        int k=0;
        for(int i=0;i<nums.length;i++){
          if(nums[i]!=val){
            nums[k++]=nums[i];
          }
        }
        return k;

    }

}